hasher-matcher-actioner/terraform/lcc_custodian/main.tf

```diff
@@ -113,6 +113,53 @@ data "aws_iam_policy_document" "custodian" {
 
 }
 
+
+locals {
+
+  # all_lambda_names = flatten([[for lambda in var.pipeline_lambdas : lambda[1]], var.other_lambdas])
+  title_glance = jsonencode({
+    height = 6,
+    width  = 6,
+    type   = "metric",
+    properties = {
+      view    = "timeSeries",
+      stacked = false,
+      metrics = [["ThreatExchange/HMA", "lcc.build_index-count"]]
+      # metric = [for lambda in local.all_lambda_names : ["ThreatExchange/HMA", "lcc.build_index-count"]]
+      region = "${data.aws_region.current.name}"
+    }
+  })
+
+
+  first_item = jsonencode({
+    width = 12,
+    type  = "metric",
+    properties = {
+      metrics = [["ThreatExchange/HMA", "lcc.build_index-duration"]]
+      view    = "timeSeries",
+      stacked = false,
+      region  = "${data.aws_region.current.name}"
+      period  = 300
+      stat    = "Average"
```

Comment:
Average stats for duration does not add a lot of value, use ` stat = "p90"` instead.